conf/solr/crawldb/conf/managed-schema

```diff
@@ -116,20 +116,48 @@
    <!-- Crawl DB Static Fields -->
    <field name="id" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" /> 
    <field name="url" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" />
-   <field name="jobId" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" />
    <field name="group" type="string" indexed="true" stored="true" required="true"  multiValued="false" docValues="true" /> 
-   <field name="lastFetchedAt" type="date" indexed="true" stored="true" multiValued="false" docValues="true" />
-   <field name="numTries" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
-   <field name="numFetches" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
-   <field name="status" type="status" indexed="true" stored="true" default="NEW" multiValued="false" required="true" docValues="true" />
-   <field name="depth" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
    <field name="score" type="double" indexed="true" stored="true" default="0" multiValued="false" docValues="true" />
-   <field name="plainText" type="text_general" indexed="true" stored="true" multiValued="false" />
 
+   <!-- Deprecated Fields -->
+   <!-- <field name="jobId" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" /> -->
+   <!-- <field name="lastFetchedAt" type="date" indexed="true" stored="true" multiValued="false" docValues="true" /> -->
+   <!-- <field name="numTries" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="numFetches" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="status" type="status" indexed="true" stored="true" default="NEW" multiValued="false" required="true" docValues="true" /> -->
+   <!-- <field name="depth" type="int" indexed="true" stored="true" default="0" multiValued="false" docValues="true" /> -->
+   <!-- <field name="plainText" type="text_general" indexed="true" stored="true" multiValued="false" /> -->
+
+   <!-- Proposed Sparkler Schema -->
+   <field name="fetch_timestamp" type="date" indexed="true" stored="true" multiValued="false" docValues="true" /> 
+   <field name="content_type" type="string" indexed="true" stored="true" multiValued="false" docValues="true" /> 
+   <field name="crawler" type="string" indexed="true" stored="true" multiValued="false" default="sparkler"/> 
+   <field name="crawl_id" type="string" indexed="true" stored="true" required="true" multiValued="false" docValues="true" /> 
+   <field name="raw_content" type="text_general" indexed="true" stored="true" multiValued="false" />
```

Comment:
Multiple reasons for the snake case:
- CDR Schema
- Better readability
- Acknowledge your concern about `_` char but wouldn't it be more applicable where we have a long list of fields?
- Also, as far as I am aware, Solr follows snakecase for field name.